pcsx2/ps2/pgif.cpp

```diff
@@ -392,13 +392,13 @@ u32 immRespHndl(u32 cmd, u32 data)
         case 0x01:
             break;  //Returns Nothing (old value in GPUREAD remains unchanged)
         case 0x02:
-            data = pgif1reg & 0x001FFFFF;
+            data = pgif1reg & 0x000FFFFF;
             break;  //Read Texture Window setting  ;GP0(E2h) ;20bit/MSBs=Nothing
         case 0x03:
-            data = pgif2reg & 0x001FFFFF;
+            data = pgif2reg & 0x00FFFFF;
```

Comment:
Just an annoyance, but if you're gonna pad it with zero's can you make sure it's of even length (just needs one more zero)